edx_sga/tests/integration_tests.py

```diff
@@ -2,49 +2,60 @@
 import os
 import shutil
 import tempfile
-import urllib
 
-from ddt import ddt, data, unpack
 import mock
-from opaque_keys.edx.locator import CourseLocator
-from opaque_keys.edx.locations import Location  # lint-amnesty, pylint: disable=import-error
-import pytz
+import six.moves.urllib.error  # lint-amnesty, pylint: disable=import-error
+import six.moves.urllib.parse  # lint-amnesty, pylint: disable=import-error
+import six.moves.urllib.request  # lint-amnesty, pylint: disable=import-error
 
-from lms.djangoapps.courseware import module_render as render  # lint-amnesty, pylint: disable=import-error
-from lms.djangoapps.courseware.models import StudentModule  # lint-amnesty, pylint: disable=import-error
-from lms.djangoapps.courseware.tests.factories import StaffFactory  # lint-amnesty, pylint: disable=import-error
-from django.contrib.auth.models import User  # lint-amnesty, pylint: disable=import-error
-from django.core.exceptions import PermissionDenied  # lint-amnesty, pylint: disable=import-error
+import pytz
+from ddt import data, ddt, unpack
+from django.contrib.auth.models import \
+    User  # lint-amnesty, pylint: disable=import-error
+from django.core.exceptions import \
+    PermissionDenied  # lint-amnesty, pylint: disable=import-error
 from django.db import transaction  # lint-amnesty, pylint: disable=import-error
-from django.test.utils import override_settings  # lint-amnesty, pylint: disable=import-error
+from django.test.utils import \
+    override_settings  # lint-amnesty, pylint: disable=import-error
+from edx_sga.constants import ShowAnswer
+from edx_sga.sga import StaffGradedAssignmentXBlock
+from edx_sga.tests.common import (DummyResource, TempfileMixin, get_sha1,
+                                  is_near_now, parse_timestamp, reformat_xml)
+from lms.djangoapps.courseware import module_render as render  # lint-amnesty, pylint: disable=import-error
+from lms.djangoapps.courseware.models import \
+    StudentModule  # lint-amnesty, pylint: disable=import-error
+from lms.djangoapps.courseware.tests.factories import \
+    StaffFactory  # lint-amnesty, pylint: disable=import-error
+from opaque_keys.edx.locations import \
+    Location  # lint-amnesty, pylint: disable=import-error
+from opaque_keys.edx.locator import CourseLocator
+from student.models import (  # lint-amnesty, pylint: disable=import-error
+    UserProfile, anonymous_id_for_user)
+from student.tests.factories import \
+    AdminFactory  # lint-amnesty, pylint: disable=import-error
 from submissions import api as submissions_api  # lint-amnesty, pylint: disable=import-error
-from submissions.models import StudentItem  # lint-amnesty, pylint: disable=import-error
-from student.models import anonymous_id_for_user, UserProfile  # lint-amnesty, pylint: disable=import-error
-from student.tests.factories import AdminFactory  # lint-amnesty, pylint: disable=import-error
+from submissions.models import \
+    StudentItem  # lint-amnesty, pylint: disable=import-error
```

Comment:
is there a way we can add this rule to whole file.

Reply:
I disabled this rule for the entire repo due to it being disabled in most .py files.